master won't compile

As of d70e68b  from earlier today 
ADAM master won't compile for me

```
[ERROR] /home/jp/test_compile/v1/adam/adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala:297: error: not found: value resourcePath
[ERROR]     val inputPath = resourcePath("HLA_DQB1_05_01_01_02.fa")
[ERROR]                     ^
[ERROR] one error found
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] ADAM_2.10 .......................................... SUCCESS [  4.943 s]
[INFO] ADAM_2.10: Core .................................... FAILURE [ 37.906 s]
[INFO] ADAM_2.10: APIs for Java ........................... SKIPPED
[INFO] ADAM_2.10: CLI ..................................... SKIPPED
[INFO] ADAM_2.10: Assembly ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE

```

As a control, the earlier commit b903cfd
does compile fine for me

--- a/adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala
+++ b/adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala
@@ -294,7 +294,7 @@ class ADAMContextSuite extends ADAMFunSuite {
   }
 
   sparkTest("read a HLA fasta from GRCh38") {
-    val inputPath = resourcePath("HLA_DQB1_05_01_01_02.fa")
+    val inputPath = testFile("HLA_DQB1_05_01_01_02.fa")
     val gRdd = sc.loadFasta(inputPath, 10000L)
     assert(gRdd.sequences.records.size === 1)
     assert(gRdd.sequences.records.head.name === "HLA-DQB1*05:01:01:02")

